doc/Language/variables.pod6

```diff
@@ -293,8 +292,8 @@ generated for you because you used the C<.> twigil when the attributes were
 declared.  Note, however, that subclasses may override those methods. If you
 don't want this to happen, use C<$!x> and C<$!y> instead.
 
-The fact that the C<.> twigil just does a method call also implies that the
-following is possible too:
+The fact that the C<.> twigil does a method call implies that the
+following is possible, too:
```

Comment:
Okay.
The fact that the C<.> twigil does a method call implies that the
following is also possible: